### ISSUE TITLE
Add author's avatar to the individual issues

### ISSUE THREAD

**Issue by BasThomas** [SwiftWeekly/swiftweekly.github.io#464]
As per https://twitter.com/daveverwer/status/1139299506472796161 😃

We use avatars on the authors page already, so that should be a good start: https://github.com/SwiftWeekly/swiftweekly.github.io/blob/464c1e6994f3075890b7b95687ec458d12eed942/authors.html#L19

**Comment by jessesquires** [comment]
Awesome idea. 

We should make this a new `includes/` component to avoid duplicate code.
